rollup.config.js

```diff
@@ -4,6 +4,7 @@ const ts = require('rollup-plugin-typescript2')
 const replace = require('rollup-plugin-replace')
 const alias = require('rollup-plugin-alias')
 const json = require('rollup-plugin-json')
+const runtimeCorePkg = require('./packages/runtime-core/package.json')
```

Comment:
Should be using `version` from `lerna.json` which is the source of truth.

Reply:
@yyx990803 completed!